command/namespace_status_test.go

```diff
@@ -52,6 +54,7 @@ func TestNamespaceStatusCommand_Good(t *testing.T) {
 	// Create a namespace
 	ns := &api.Namespace{
 		Name: "foo",
+		//Description: "Test namespace",
```

Comment:
If this isn't being used, can we remove it?

Reply:
It can, my bad :)